src/Runners/PHPUnit/Options.php

```diff
@@ -115,6 +156,18 @@ public function __get($var)
     }
 
     /**
+     * Public read accessibility
+     * (e.g. to make empty($options->property) work as expected)
+     *
+     * @param string $var
+     * @return mixed
+     */
+    public function __isset($var)
```

Comment:
It just looks inconsistent. I would either expect an ArrayAccess implementation with exceptions on set or real implemented getters but that might be personal preference and is definitely out of scope of this PR so nvm